app/src/main/java/chat/rocket/android/profile/presentation/ProfilePresenter.kt

```diff
@@ -29,39 +30,42 @@ class ProfilePresenter @Inject constructor (private val view: ProfileView,
                 myselfId = myself.id
                 val avatarUrl = UrlHelper.getAvatarUrl(serverUrl, myself.username!!)
                 view.showProfile(
-                    avatarUrl,
-                    myself.name!!,
-                    myself.username!!,
-                    myself.emails?.get(0)?.address!!
+                        avatarUrl,
+                        myself.name!!,
+                        myself.username!!,
+                        myself.emails?.get(0)?.address!!
                 )
             } catch (exception: RocketChatException) {
                 exception.message?.let {
                     view.showMessage(it)
                 }.ifNull {
-                        view.showGenericErrorMessage()
-                    }
+                            view.showGenericErrorMessage()
```

Comment:
Could you fix this wrong indentation? I know that when using `.ifNull {` the IDE is putting the wrong indentation, but we should indent this correct (in this case aligned with `view.showMessage(it)`)